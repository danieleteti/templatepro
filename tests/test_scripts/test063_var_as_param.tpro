{{if intvalue10|eq,intvalue10}}
intvalue10 = intvalue10
{{endif}}
x{{:value2|lpad,intvalue10}}x

{{:intvalue10|eq,dataitem.PropInt}}
{{:intvalue10|gt,dataitem.PropInt}}
{{:intvalue10|ge,dataitem.PropInt}}
{{:intvalue10|lt,dataitem.PropInt}}
{{:intvalue10|le,dataitem.PropInt}}

When Null is involved in any kind of expression
result is always Null

{{:intvalue10|eq,dataitemnullable.NullInt32}}