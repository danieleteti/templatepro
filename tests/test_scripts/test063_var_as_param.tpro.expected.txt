intvalue10 = intvalue10
x  DANIELE2x

False
True
True
False
False

When Null is involved in any kind of expression
result is always Null

False